awx/ui/client/src/login/loginModal/loginModal.partial.html

```diff
@@ -23,7 +23,7 @@
                         Your session timed out due to inactivity. Please sign in.
                     </div>
                 </div>
-                <div class="LoginModal-alert LoginModal-alert--error" ng-show="userLoggedOut">
+                <div class="LoginModal-alert LoginModal-alert--error" ng-show="userLoggedOut && !sessionExpired">
```

Comment:
I would suggest using separate `ng-show` expressions for the two different logout messages. The current expression will return truthy if one of those criteria are met.